xbout/load.py

```diff
@@ -8,6 +8,9 @@
 
 from natsort import natsorted
 
+_bout_timing_variables = ['wall_time', 'wtime', 'wtime_rhs', 'wtime_invert',
```

Comment:
Conventionally names of global variables should be in all caps, and this is what I did in other parts of the code (the front underscore is still appropriate).